istio-install/overlays/add-fluent-bit.yaml

```diff
@@ -0,0 +1,44 @@
+#@ load("@ytt:overlay", "overlay")
+
+#@overlay/match by=overlay.subset({"kind": "Deployment", "metadata":{"name":"istio-ingressgateway"}}),expects=1
```

Comment:
This is based on what we've beign  [here](https://github.com/cloudfoundry/cf-k8s-networking/blob/c37068a86565bcb81c0ea42891c613ae7006780c/istio-install/overlays/ingre
ssgateway-daemonset.yaml#L3) for the istio-ingressgateway, if we were to do that we would do it for all of them.

Reply:
I believe this is based on what we do for all the [overlays](https://github.com/cloudfoundry/cf-k8s-networking/blob/c37068a86565bcb81c0ea42891c613ae7006780c/istio-install/overlays/ingre
ssgateway-daemonset.yaml#L3) that edit the ingressgateway. I think we can make this configurable but maybe it would be better to do this in another story.

Reply:
I wonder if we can make it match DaemonSet _or_ Deployment? Otherwise couldn't this fail depending on the order that overlays get applied in?

Reply:
Yeah, they were being applied alphabetically which is why we made it `Deployment`.